src/database.cpp

```diff
@@ -19,11 +19,28 @@ static tfs::detail::Mysql_ptr connectToDatabase(const bool retryIfError)
 	}
 	isFirstAttemptToConnect = false;
 
+// MariaDB requires explicit SSL settings to avoid the following error:
+// "SSL is required, but the server does not support it"
+// For more details see issue #4954 ( https://github.com/otland/forgottenserver/issues/4954 )
+#ifdef MARIADB_VERSION_ID
```

Comment:
Do we need to guard it? What happens if you let this code build against libmysql?